### DISCUSSION TITLE
ExhibitionEvent modelling

### DISCUSSION THREAD

**Discussion by GenFab28** [culturecreates/artsdata-planet-lavitrine#7] [Modelling discussions]
The type ExhibitionEvent will not have the relation with an offer
How can we configure the model to make the distinction between an EventSeries +Event and an Exhibition event type on PDA

**Comment by saumier** [comment]
@GenFab28 I can configure the "Frame" to not include 'offers' in the data feed when the Event type is "ExhibitionEvent".  Did I understand your question correctly?

**Reply by GenFab28** [reply]
@saumier Yes, you understand correctly and it perfect.

**Comment by saumier** [comment]
Created issue https://github.com/culturecreates/artsdata-planet-lavitrine/issues/11 to track implementation.
